helpers/inbound/inbound.go

```diff
@@ -42,40 +41,41 @@ func indexHandler(response http.ResponseWriter, request *http.Request) {
 }
 
 func getBoundary(value string, contentType string) (string, *strings.Reader) {
```

Comment:
I don't think this function is necessary anymore but I'm unsure about taking it out. Thoughts?